src/client/pages/OfferNew/Checkout/SignDisclaimer.tsx

```diff
@@ -23,61 +21,51 @@ const Wrapper = styled('div')`
     margin: 0;
   }
 `
+const getSignButtonLabel = (textKeys: TextKeyMap, signMethod?: SignMethod) => {
+  return signMethod === SignMethod.SimpleSign
+    ? textKeys.CHECKOUT_SIMPLE_SIGN_BUTTON_TEXT()
+    : textKeys.CHECKOUT_SIGN_BUTTON_TEXT()
+}
+
+const getPrivacyPolicyLink = (currentLocale: string, quote: OfferQuote) => {
+  const linkFromBackend = quote.insuranceTerms.get(
+    InsuranceTermType.PrivacyPolicy,
+  )?.url
+
+  if (linkFromBackend) {
+    return linkFromBackend
+  }
+
+  return `https://www.hedvig.com/${currentLocale}/privacy`
+}
```

Comment:
As we talked about it would be best to get this from BE in the end, since the URL should be localised

Reply:
Yeah I know this is not perfect - but it works as a temporary solution. And as long as we add redirects in Storyblok when url's are localised we're kind of OK? Maybe 😄